packages/trees/frontend/sirius-components-trees/src/treeitems/context-menu/DeleteMenuItem.tsx

```diff
@@ -37,6 +37,19 @@ const deleteTreeItemMutation = gql`
   }
 `;
 
+export const handleDeleteItem = (editingContextId, treeId, item, deleteTreeItem, showDeletionConfirmation, onClick) => {
```

Comment:
There's no need to share this part of the code and create additional coupling between those components, the hook will be enough.